platform/jvm/rules.bzl

```diff
@@ -172,7 +172,7 @@ windowsWiXToolsetPath: {}
     )
 
     config_path_arg = "--config_path={}".format(config_file.path)
-    if "APPLE_CODE_SIGN" in ctx.var and ctx.var["APPLE_CODE_SIGN"].lower() in "yes, true":
+    if "APPLE_CODE_SIGN" in ctx.var and ctx.var["APPLE_CODE_SIGN"] == "yes":
```

Comment:
Given that we do need the `config_setting` where `APPLE_CODE_SIGN` is specifically set to "yes", we also need to tighten up this check to align with it.